packages/truffle/test/sources/migrations/typescript/migrations/global.d.ts

```diff
@@ -0,0 +1,4 @@
+export {};
+declare global {
+  const artifacts: any;
+}
```

Comment:
This file is just a temporary hack for now - this won't be necessary in user code when this is finished